lib/enigma.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './encryptor'
+require_relative './decryptor'
+
+class Enigma
+
+  def encrypt(message, key ='', date='')
+    encryptor = Encryptor.new(message, key, date)
```

Comment:
the `key` and `date` arguments are also optional in the initialize method of the `Encryptor` class.